### ISSUE TITLE
Update Jackson to 2.8.3

### ISSUE THREAD

**Issue by dadoonet** [dadoonet/legacy-search#38]

**Comment by dadoonet** [comment]
This can't really be done ATM

```
Exception in thread "main" java.lang.NoSuchMethodError: org.apache.logging.log4j.core.impl.ThrowableProxy.getExtendedStackTraceAsString(Ljava/util/List;Lorg/apache/logging/log4j/core/pattern/TextRenderer;)Ljava/lang/String;
    at org.apache.logging.log4j.core.pattern.ExtendedThrowablePatternConverter.format(ExtendedThrowablePatternConverter.java:68)
    at org.apache.logging.log4j.core.pattern.PatternFormatter.format(PatternFormatter.java:38)
    at org.apache.logging.log4j.core.layout.PatternLayout$PatternSerializer.toSerializable(PatternLayout.java:294)
    at org.apache.logging.log4j.core.layout.PatternLayout.toText(PatternLayout.java:195)
    at org.apache.logging.log4j.core.layout.PatternLayout.encode(PatternLayout.java:180)
    at org.apache.logging.log4j.core.layout.PatternLayout.encode(PatternLayout.java:57)
    at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.directEncodeEvent(AbstractOutputStreamAppender.java:176)
    at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.tryAppend(AbstractOutputStreamAppender.java:169)
    at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.append(AbstractOutputStreamAppender.java:160)
    at org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
    at org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
    at org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
    at org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
    at org.apache.logging.log4j.core.config.LoggerConfig.callAppenders(LoggerConfig.java:447)
    at org.apache.logging.log4j.core.config.LoggerConfig.processLogEvent(LoggerConfig.java:432)
    at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:416)
    at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:402)
    at org.apache.logging.log4j.core.config.AwaitCompletionReliabilityStrategy.log(AwaitCompletionReliabilityStrategy.java:63)
    at org.apache.logging.log4j.core.Logger.logMessage(Logger.java:146)
    at org.apache.logging.slf4j.Log4jLogger.log(Log4jLogger.java:376)
    at org.eclipse.jetty.util.log.JettyAwareLogger.log(JettyAwareLogger.java:607)
    at org.eclipse.jetty.util.log.JettyAwareLogger.warn(JettyAwareLogger.java:431)
    at org.eclipse.jetty.util.log.Slf4jLog.warn(Slf4jLog.java:69)
    at org.eclipse.jetty.util.component.AbstractLifeCycle.setFailed(AbstractLifeCycle.java:204)
    at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:74)
    at restx.server.JettyWebServer.start(JettyWebServer.java:109)
    at restx.server.JettyWebServer.startAndAwait(JettyWebServer.java:114)
    at fr.pilato.demo.legacysearch.app.LegacySearchApp.main(LegacySearchApp.java:39)
```

I suspect that RESTX is causing some troubles here.

**Comment by dadoonet** [comment]
Could be similar to https://github.com/restx/restx/pull/228 so may be I need also to send a new PR to RestX project.

**Comment by dadoonet** [comment]
Fixed by #55
